cdc/sink/manager/kafka/manager_test.go

```diff
@@ -0,0 +1,112 @@
+// Copyright 2022 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package kafka
+
+import (
+	"testing"
+	"time"
+
+	kafkaconfig "github.com/pingcap/tiflow/cdc/sink/producer/kafka"
+	kafkamock "github.com/pingcap/tiflow/pkg/kafka"
+	"github.com/stretchr/testify/require"
+)
+
+func TestPartitions(t *testing.T) {
+	t.Parallel()
+
+	client := kafkamock.NewClientMockImpl()
+	adminClient := kafkamock.NewClusterAdminClientMockImpl()
+	defer func(adminClient *kafkamock.ClusterAdminClientMockImpl) {
+		_ = adminClient.Close()
+	}(adminClient)
+	cfg := &kafkaconfig.AutoCreateTopicConfig{
+		AutoCreate:        true,
+		PartitionNum:      2,
+		ReplicationFactor: 1,
+	}
+
+	manager := NewTopicManager(client, adminClient, cfg)
+	partitionsNum, err := manager.Partitions(kafkamock.DefaultMockTopicName)
+	require.Nil(t, err)
+	require.Equal(t, int32(3), partitionsNum)
```

Comment:
Does it mean that `parititionsNum` is 3 in the beginning, but after 1min, it will be changed to 2?

Reply:
No, this is just a test of its initial value.